src/iiif.py

```diff
@@ -5,7 +5,7 @@
 from cache_requests import install_cache_requests
 import conf, utils
 
-LOG = logging.getLogger(__name__)
+LOG = conf.multiprocess_log('iiif.log', __name__)
```

Comment:
at least locally, this logger seemed to be configured not to write to a file nor stderr